### PR TITLE
Add sample application with built in db:seed task

### PR DESCRIPTION
**Pull request by jayroh**
For things like testing the web driving, it is required that we have a
"finished" application that can be used for that work.

This PR adds sample data that came from one end-to-end
submission with the bare minimum. With regards to the data - nils, empty
arrays, or anything that was "barren" was removed.

Also included - a snap application with a more complete profile.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1 +1,61 @@
 # frozen_string_literal: true
+
+puts "Creating test snap application..."
+
```
Comment (zspencer) [inline db/seeds.rb]: _not blocking_ - Does this make sense to make a factory?

Comment (jayroh) [reply db/seeds.rb]: You *could* do that, sure. I don't have any strong feelings towards making this use factory girl. However, that would require pulling in FG to the development group, and maintaining it over in that context is a jump away.

In truth, I have no argument for or against it. What about you, @zspencer?

**Comment by jayroh** [comment]
> Thoughts on potentially adding the other half (a "full" application, which would have multiple members, different employment statuses etc)

Yup. I can do that.

**Comment by jayroh** [comment]
[Easy way to create a minimal or full 1171 application in order to do QA](https://trello.com/c/DWz25eGR/185-easy-way-to-create-a-minimal-or-full-1171-application-in-order-to-do-qa)

**Comment by jayroh** [comment]
@jessieay I'm not sure *how* complete we want it but my latest commit adds another snap application that has multiple members, multiple addresses, more answers.

File: db/seeds.rb (hunk 1 above)
Comment (zspencer) [reply db/seeds.rb]: Yeah I'm fine leaving it as a normal model. One of the things I was thinking about for testing was to add to the admin dashboard  SNAPApplication generator which could use factory girl traits and what not to build an app; but I don't think that's necessary for now.

**Pull request closed without merging**